server/http/src/server.rs

```diff
@@ -158,3 +167,60 @@ impl GraphQLServerTrait for GraphQLServer {
         Ok(Box::new(task))
     }
 }
+
+#[test]
+fn emits_a_combined_schema_after_one_schema_is_added() {
```

Comment:
I guess the test name is no longer accurate since we're not combining schemas "anymore" and won't be.